tests/test_history.py

```diff
@@ -1,96 +1,103 @@
 """Tests the xonsh history."""
+# pylint: disable=protected-access
 from __future__ import unicode_literals, print_function
 import io
 import os
 import sys
 
 import nose
-from nose.tools import assert_equal, assert_true
+from nose import tools as nose_tools
```

Comment:
FWIW, If the primary maintainers feel strongly about the Google import style rule that I've employed in this change I'm willing to revert those aspects. I'd much rather see the code be lint clean than fight for a change in the style of the code. On the other hand if I've managed to convince you the Google rule is better then I intend to apply it to all of the other modules as I work my way through the code removing lint errors so that there is a consistent style.